src/features/author/authorSlice.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Consider removing 'eslint-disable no-param-reassign' and avoid modifying the function parameters directly. Instead, copy them to new variables and modify them if needed.